packages/block-editor/src/components/justify-toolbar/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup } from '@wordpress/components';
+import {
+	justifyLeft,
+	justifyCenter,
+	justifyRight,
+	justifySpaceBetween,
+} from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+const navIcons = {
```

Comment:
This should be renamed to something more generic.

Reply:
:+1: Addressed in b68cf5d733